scripts/collector/mysql/collect-data.sh

```diff
@@ -119,11 +119,11 @@ function checkVersionPg {
     fi
 
     # SELECT 'DMAFILETAG~' , version();
-    dbversion=$(PGPASSWORD="$pass" ${SQLCMD}  --user=$user -d $db -h $host -w -p $port -t --no-align << EOF
+    dbVersion=$(PGPASSWORD="$pass" ${SQLCMD} -X --user=$user -d $db -h $host -w -p $port -t --no-align << EOF
 SELECT current_setting('server_version_num');
 EOF
 )
-echo 'DMAFILETAG~'${dbversion}'|'${dbversion}'_'${OpVersion}'_'${host}'-'${port}'_'${db}'_'${db}'_'$(date +%y%m%d%H%M%S)
+echo 'DMAFILETAG~'${dbVersion}'|'${dbVersion}'_'${OpVersion}'_'${host}'-'${port}'_'${db}'_'${db}'_'$(date +%y%m%d%H%M%S)
 }
 
 function checkVersionMysql {
```

Comment:
Should these edits go to a different PR?

Reply:
well, i guess this should only move if it's not part of the mysql updates.  Now that I look at it, it may be part of the PG section.

Reply:
Actually, all the changes in the MySQL directory need to go into another PR.